docs/guides/basics/parameters.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Parameters
+---
+
+# Accept input with parameters
+
+The primary mechanism for passing data from the user or document into your Pack is via parameters. You define the parameters in your code and the user fills them with values when they use your Pack. The same parameters are used for formulas, actions, and sync tables.
```

Comment:
Slightly confusing what "the same parameters means", maybe instead something about the same mechanism for specifying parameter definitions.

Reply:
Done